reviews/sandbox/2022-wasmedge-annual.md

```diff
@@ -0,0 +1,155 @@
+# WasmEdge Runtime-2022-Annual Review
+
+## Background
+
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime. The ultimate goal for WasmEdge is to bring cloud-native and serverless application paradigms to edge, streaming, web3, and SaaS computing environments.
+
+The unique features and advantages of WasmEdge include the following.
+
+* One of the [fastest](https://github.com/WasmEdge/WasmEdge#performance) WebAssembly VMs on the market (based on **LLVM AOT**)
```

Comment:
Suggestion: Adding link for WASM Runtime benchmarks for 2023 https://00f.net/2023/01/04/webassembly-benchmark-2023/ might be a bit more relevant in comparison to the ones mentioned in the readme, which are from 2021. 
(Note: this is a specific user's feedback on the performance if the community feels it is not reliable. it can be ignored.)